microsoft-edge/hosting/webview2/Tooling/WebDriver.md

```diff
@@ -0,0 +1,117 @@
+
+## Step 3: Setup Project
+At this point you should have Microsoft Edge installed, have built a WebView2 project, and have installed WebDriver. Now, let’s get started using WebDriver.
+
+1. Start by creating a new **C# .NET Framework** project in **Visual Studio**.
+
+![alt text](..\images\WebDriver\NewProject.PNG)
+
+2. Click **Next** on the bottom right hand corner to continue.
+
+![alt text](..\images\WebDriver\App-Create.PNG)
+
+3. Give your project a **name**, save it to your preferred **location**, and click **Create**.
+
+![alt text](..\images\WebDriver\Start-App.PNG)
+
+4. Now let's add **WebDriver** to the project. You can install WebDriver via the **Selenium.WebDriver NuGet package**. 
+
+To download the **Selenium.WebDriver NuGet package**, in **Visual Studio**, hover over **Project** and select **Manage NuGet Package**. The following screen should appear:
+
+![alt text](..\images\WebDriver\Download-Nuget.PNG)
+
+5. Enter **Selenium.WebDriver** in the search bar, click **Selenium.WebDriver** from the results, and make sure to checkmark the box next to **include pre-release**. On the right hand side window, ensure the **Version** is set to **install 4.0.0-alpha04** and click **Install**. Nuget will download WebDriver to your machine.
+
+To learn more about the **Selenium.WebDriver NuGet package** click [here](https://www.nuget.org/packages/Selenium.WebDriver/4.0.0-alpha04)
+
+![alt text](..\images\WebDriver\Nuget.PNG)
+
+6. Use **OpenQA.Selenium.Edge** by adding the following using statement:```using OpenQA.Selenium.Edge;``` to the top of the file.
+
+```C#
+using OpenQA.Selenium.Edge;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+```
+
+## Step 4: Setup WebDriver
+
+1. First, create the **EdgeOptions** object, by copying the code below. The **EdgeOptions** object takes in two parameters:
+    1. **is_legacy** is set to false, indicating you are using the new Chromium-powered Microsoft Edge Browser.
+    2. The **"webview2"** string specifies that the **WebDriver** is driving **WebView2**.
+
+
+```C#
+static void Main(string[] args)
+{
+    // EdgeOptions() requires using OpenQA.Selenium.Edge
+    // Construct EdgeOptions with is_legacy = false and the string "webview2"
+    EdgeOptions edgeOptions = new EdgeOptions(false, "webview2");
+```
+
+2. Next, set **edgeOptions BinaryLocation** to the filepath of your WebView2 project's executable, create a string to store the path to **EdgeDriver's** executable, and create a string to store the name of the **EdgeDriver**. Use these two strings to construct the **EdgeDriverService** object as shown below. Finally, create the **EdgeDriver** object using **EdgeDriverService** and **EdgeOptions**.
+
+You can copy and paste the following code underneath **edgeOptions**. Make sure to specify the correct file paths to your project's executable and the edgedriver's executable on your machine.
```

Comment:
Again, I would make this `edgeOptions` instead of **edgeOptions**. Also, this should be: "and the Microsoft Edge Driver's executable on your machine."